csibe.py

```diff
@@ -186,6 +220,11 @@ def submodule_init_and_update(repository_path):
         action="store_true",
         help="turn on debug mode")
 
+    parser.add_argument(
+        "--old-testbed",
+        action="store_true",
+        help="download and run the benchmarks on the old testbed")
```

Comment:
Currently this option downloads and exists. The help message should reflect this.